hashicorp/dev-portal
# chore(deps): bump marketo-form

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-BRANCH_NAME-hashicorp.vercel.app/PATH_TO_VIEW) 🔎
- [Asana task](url) 🎟️

## 🗒️ What

This PR updates `@hashicorp/react-marketo-form` to support the new privacy policy disclaimer.

## 🤷 Why

Marketing Operations has adjusted their desired privacy policy stance to be both GDPR-compliant and allow implicit privacy policy acceptance in the United States.

## 🛠️ How

- Updated the `@hashicorp/react-marketo-form` dependency
- Updated the in-tree data for the Marketo form
- Adjust styling to match dev portal's design.

## 📸 Design Screenshots



## 🧪 Testing

- Go to certifications page
- Ensure newsletter form contains the "By submitting this form,..." text
- Ensure styling adapts to both light and dark modes

## 💭 Anything else?

This PR can be reviewed, but should not be merged until we release the production version of `@hashicorp/react-marketo-form` (sometime next week).
